cfgov/v1/preprocessed/js/modules/classes/BreakpointHandler.js

```diff
@@ -72,7 +70,7 @@ function _init( self ) {
  */
 function watchWindowResize() {
   var self = this;
-  $( window ).bind( 'resize', function() {
+  window.addEventListener( 'resize', function() {
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/ec5ce18e59292f3b42e8578e3cd4470b2ed8d1a8